### ISSUE TITLE
Radios in radio groups broken in IE 11

### ISSUE THREAD

**Issue by smares** [vuetifyjs/vuetify#1588]
### Steps to reproduce
Create a radio group and fill it with at least two radios, then look at the outcome in IE 11

### Versions
Vue 2.4.2, Vuetify 0.15.2, IE 11

### What is expected ?
The radios should be displayed one under the other with enough spacing between

### What is actually happening ?
The radios are squeezed together and overlap

### Reproduction Link
https://codepen.io/smares/pen/ayMMwj

![image](https://user-images.githubusercontent.com/818380/30033651-b0a383f2-919c-11e7-8208-8ad3b5f836d6.png)

**Comment by jacekkarczmarczyk** [comment]
In discord we've found 2 solutions: 

1. by @Kaelwd - fixes IE flexbox bug (#1 on the list https://github.com/philipwalton/flexbugs)
```
.input-group {
-  flex: 1;
+  flex: 1 1 auto;
}
```

2. by me - removes `display: flex` from radio buttons as it's seems theat flex is not necessary here (all children are aboslutely positioned or hidden)
```
.radio-group--column .input-group__input {
    display: block
}
```

**Comment by johnleider** [comment]
The flex is required as radio-group can show the radios in row or column.
